extensions/dust-tt/package.json

```diff
@@ -8,52 +8,104 @@
   "contributors": [
     "vineus",
     "henry_fontanier",
-    "stang"
+    "stang",
+    "fraggle"
   ],
   "categories": [
     "Productivity"
   ],
   "license": "MIT",
   "preferences": [
     {
-      "name": "apiBaseUrl",
-      "description": "The base URL of the Dust API",
+      "name": "connexionFlow",
+      "description": "Specify how to connect to Dust",
+      "type": "dropdown",
+      "required": true,
+      "title": "How to connect to Dust ?",
+      "data": [
+        {
+          "value": "oauth",
+          "title": "Sign in with your Dust account"
+        },
+        {
+          "value": "apiKey",
+          "title": "Use workspace API key (legacy)"
+        }
+      ]
+    },
+    {
+      "name": "apiUrl",
+      "description": "Dust Api URL (advanced)",
       "type": "textfield",
-      "required": false,
-      "title": "API URL",
-      "default": "https://dust.tt/api/v1/w",
-      "placeholder": "https://dust.tt/api/v1/w"
+      "required": true,
```

Comment:
I'm not 100% sure what happen if required if false but there is a default value ?

(I don't **need** to show it to the user but I need a value)